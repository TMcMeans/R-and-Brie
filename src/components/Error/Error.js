import React from 'react'

export const Error = () => {

  return (
    <div>
      <h1>You are in a part of the site not found.</h1>
    </div>
  )
}
