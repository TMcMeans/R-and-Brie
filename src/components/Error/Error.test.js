import React from 'react'
import { shallow } from 'enzyme'

import { Error } from './Error'

describe('Error', () => {
  it('should match snapshot with correct data passed in', () => {
    const wrapper = shallow(<Error />)
    expect(wrapper).toMatchSnapshot()
  })
})