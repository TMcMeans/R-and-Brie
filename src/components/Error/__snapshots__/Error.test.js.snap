// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should match snapshot with correct data passed in 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Error />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        You are in a part of the site not found.
      </h1>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "You are in a part of the site not found.",
      },
      "ref": null,
      "rendered": "You are in a part of the site not found.",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          You are in a part of the site not found.
        </h1>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "You are in a part of the site not found.",
        },
        "ref": null,
        "rendered": "You are in a part of the site not found.",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
